From 514aef508bd56fa5f03a27662b4c4a86f4f1c80f Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Sun, 11 Jun 2023 19:39:57 +0200
Subject: [PATCH 380/463] arm64: dts: rk3399-pinephone-pro: Add I2C supply to
 stk3311

It is needed to communicate with the device, when other users of
vcc3v0_touch shut down the regulator.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index 0d23cc274f219..10f5ce36d14af 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -719,6 +719,7 @@ light-sensor@48 {
 		pinctrl-0 = <&stk3311_int>;
 		vdd-supply = <&vcc_3v0>;
 		leda-supply = <&vcc_3v0>;
+		i2c-supply = <&vcc3v0_touch>;
 	};
 };
 
-- 
2.46.0

